.circleci/config.yml

```diff
@@ -343,7 +289,29 @@ jobs:
             echo "password: $CONBENCH_PASSWORD" >> .conbench
             echo "host_name: CircleCI-runner" >> .conbench
 
-            conbench cpp-micro --run-name="commit: $(git rev-parse HEAD)" --run-reason="commit"
+            # figure out run name/reason
+            if [[ $CIRCLE_BRANCH == 'main' ]]; then
+              run_name="commit: $(git rev-parse HEAD)"
+              run_reason='commit'
+            else
+              run_name="pull request $CIRCLE_PR_NUMBER: $(git rev-parse HEAD)"
+              run_reason='pull request'
+            fi
+
+            conbench cpp-micro \
+              --run-name="$run_name" \
+              --run-reason="$run_reason" \
+              --output-dir benchmark-basic-dumps/target
+
+            ls -al benchmark-basic-dumps/target
+      - run:
+          name: "Benchmarks Summary"
+          command: |
+            # We don't want to fail the CI job yet, until we ensure the numbers are stable.
+            ls -al benchmark-basic-dumps/target
+            scripts/benchmark-runner.py compare \
+                --baseline-dump-path benchmark-basic-dumps/baseline \
+                --target-dump-path benchmark-basic-dumps/target || true
```

Comment:
This will all be replaced by pulling in https://github.com/conbench/benchalerts yeah?

Reply:
Yep!